CHANGES/558.bugfix

```diff
@@ -0,0 +1,2 @@
+Fixed a bug that disallowed users who were authenticated by a remote webserver to access the
```

Comment:
can you add a line about this in our docs? I think in the authentication.rst would be good?

Reply:
Sure, I extracted the whole section to the main index.